packages/find-and-replace/FindAndReplaceManager.js

```diff
@@ -209,20 +219,24 @@ class FindAndReplaceManager {
   }
 
   _propagateUpdate(renderSelection) {
+    // HACK: we make commandStates dirty in order to trigger re-evaluation
+    this._updateMarkers()
+    this.editorSession._setDirty('commandStates')
+    if (!renderSelection) {
+      this.editorSession.skipNextSelectionRerender()
```

Comment:
Ok. I was just concerned that you may end up trapped if some implementer forgets to unblur right after the performed action.